static/sass/_pattern-font-tester.scss

```diff
@@ -0,0 +1,21 @@
+@mixin design-p-font-tester {
```

Comment:
almost direct rip [live site](https://github.com/canonical/design.ubuntu.com/blob/main/sass/_pattern_font-tester.scss), changes are on lines 12-13 and were included to appease linter